src/System.Runtime.CompilerServices.VisualC/ref/System.Runtime.CompilerServices.VisualC.cs

```diff
@@ -70,4 +73,68 @@ public sealed partial class RequiredAttributeAttribute : System.Attribute
         public RequiredAttributeAttribute(System.Type requiredContract) { }
         public System.Type RequiredContract { get { return default(System.Type); } }
     }
+    public static partial class IsPinned
+    {
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(8))]
+    public sealed partial class HasCopySemanticsAttribute : System.Attribute
+    {
+        public HasCopySemanticsAttribute() { }
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(16))]
+    public sealed partial class ScopelessEnumAttribute : System.Attribute
+    {
+        public ScopelessEnumAttribute() { }
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(4))]
+    [System.Runtime.InteropServices.ComVisibleAttribute(true)]
+    public partial class CompilerGlobalScopeAttribute : System.Attribute
+    {
+        public CompilerGlobalScopeAttribute() { }
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(1))]
+    public sealed partial class DefaultDependencyAttribute : System.Attribute
+    {
+        public DefaultDependencyAttribute(System.Runtime.CompilerServices.LoadHint loadHintArgument) { }
+        public System.Runtime.CompilerServices.LoadHint LoadHint { get { return default(System.Runtime.CompilerServices.LoadHint); } }
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(1), AllowMultiple=true)]
+    public sealed partial class DependencyAttribute : System.Attribute
```

Comment:
Only ones I told you to put in here are IsPinned, HasCopySemanticsAttribute, ScopelessEnumAttribute. You didn't ask me about the others. 

DefaultDependencyAttribute, DependencyAttribute, DiscardableAttribute,  LoadHint, StringFreezingAttribute, RuntimeWrappedException  -> System.Runtime

IUnknownConstantAttribute  -> System.Runtime.InteropServices